components/eam/src/physics/crm/crm_physics.F90

```diff
@@ -1306,16 +1477,12 @@ subroutine crm_surface_flux_bypass_tend(state, cam_in, ptend)
                            lu=.false., lv=.false., ls=.true., lq=lq)
 
    ! apply fluxes to bottom layer
-   do ii = 1,ncol
-      g_dp = gravit * state%rpdel(ii,pver)             ! note : rpdel = 1./pdel
-      ptend%s(ii,:)   = 0.
-      ptend%q(ii,:,1) = 0.
-#ifdef MAML
-      ptend%s(ii,pver)   = g_dp * cam_in%shf(ii,1)
-#else
-      ptend%s(ii,pver)   = g_dp * cam_in%shf(ii)
-#endif
-      ptend%q(ii,pver,1) = g_dp * cam_in%cflx(ii,1)
+   do icol = 1,ncol
+      g_dp = gravit * state%rpdel(icol,pver)             ! note : rpdel = 1./pdel
+      ! ptend%s(icol,:)   = 0.
```

Comment:
This now assumes that tendencies are initialized to zero in `physics_ptend_init`?

Reply:
These are always initialized to zero in the ptend_init routine, so I can delete these lines.